sim/task.h

```diff
@@ -111,7 +116,7 @@ struct TaskStatus_t {
   uint32_t ulRunTimeCounter;      /* The total run time allocated to the task so far, as defined by the run time stats clock.  See http://www.freertos.org/rtos-run-time-stats.html.  Only valid when configGENERATE_RUN_TIME_STATS is defined as 1 in FreeRTOSConfig.h. */
   //StackType_t *pxStackBase;       /* Points to the lowest address of the task's stack area. */
   uint16_t usStackHighWaterMark;  /* The minimum amount of stack space that has remained for the task since the task was created.  The closer this value is to zero the closer the task has come to overflowing its stack. */
-};
+}TaskStatus_t;
```

Comment:
nitpick: space before name

```suggestion
} TaskStatus_t;
```